### PR TITLE
Changed the default value of lane change record

### LINKED ISSUE #3341: MapBuilder: Lane change default value handling not as in open drive spec
There is a difference in the default value of the [MapBuilder](https://github.com/carla-simulator/carla/blob/dev/LibCarla/source/carla/road/MapBuilder.cpp#L198) and the [OpenDrive Spec p. 59](http://www.opendrive.org/docs/OpenDRIVEFormatSpecRev1.4H.pdf) which says for lane chane 

> allow a lane change in the indicated direction taking into account that lanes are numbered in ascending order from right to left. If the attribute is missing, “both” is assumed to be valid.

**Current behavior**

MapBuilder defaults to `none`

**Expected OpenDrive behavior**

MapBuilder defaults to `both`

### PR DESCRIPTION
**Pull request by Axel1092**
#### Description

OpenDRIVE specification (http://www.opendrive.org/docs/OpenDRIVEFormatSpecRev1.4H.pdf pg 59) requires the lane change record to be defaulted to "both" instead of "none".

Fixes #3341

#### Where has this been tested?

  * **Platform(s):** Ubuntu 18.04
  * **Python version(s):** 3
  * **Unreal Engine version(s):** 4.24

#### Possible Drawbacks
This PR needs to be tested to be sure that it does not break our towns.

---
This change is [Reviewable](https://reviewable.io/reviews/carla-simulator/carla/3361)

### CODE REVIEW

**Review by hofbi — DISMISSED** [review]

File: LibCarla/source/carla/road/MapBuilder.cpp [hunk 1]
```diff
@@ -194,8 +194,10 @@ namespace road {
       lc = RoadInfoMarkRecord::LaneChange::Decrease;
     } else if (lane_change == "both") {
```
Comment (nagasanjay) [inline LibCarla/source/carla/road/MapBuilder.cpp:195]:
```suggestion
    } else if (lane_change == "both") {
```
Did you forget to remove this line ?

File: LibCarla/source/carla/road/MapBuilder.cpp [hunk 2]
```diff
@@ -194,8 +194,10 @@ namespace road {
...
       lc = RoadInfoMarkRecord::LaneChange::Both;
```
Comment (nagasanjay) [inline LibCarla/source/carla/road/MapBuilder.cpp:196]:
```suggestion
```
Did you forget to remove this line ?

**Review by marcgpuig — APPROVED** [review]
> Reviewed 1 of 1 files at r3.
> *[Reviewable](https://reviewable.io/reviews/carla-simulator/carla/3361)* status: all files reviewed, 2 unresolved discussions (waiting on @Axel1092 and @bernatx)

**Review by Axel1092 — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/carla-simulator/carla/3361)* status: all files reviewed, 2 unresolved discussions (waiting on @bernatx and @nagasanjay)

___
*[LibCarla/source/carla/road/MapBuilder.cpp, line 195 at r1](https://reviewable.io/reviews/carla-simulator/carla/3361#-MI-L2a9eYzARami6Hfk-r1-195:-MMtdGED3WakL4TEj-hx:b-896fix) ([raw file](https://github.com/carla-simulator/carla/blob/146420e8be867a2a6b95384142eccc649f352020/LibCarla/source/carla/road/MapBuilder.cpp#L195)):*
<details><summary><i>Previously, nagasanjay (San) wrote…</i></summary><blockquote>

```suggestion
```
Did you forget to remove this line ? 
</blockquote></details>

Done.
___
*[LibCarla/source/carla/road/MapBuilder.cpp, line 196 at r1](https://reviewable.io/reviews/carla-simulator/carla/3361#-MI-L2a9eYzARami6Hfk-r1-196:-MMtdGTM1fM8qbecwNBg:b-896fix) ([raw file](https://github.com/carla-simulator/carla/blob/146420e8be867a2a6b95384142eccc649f352020/LibCarla/source/carla/road/MapBuilder.cpp#L196)):*
<details><summary><i>Previously, nagasanjay (San) wrote…</i></summary><blockquote>

```suggestion
```
Did you forget to remove this line ?
</blockquote></details>

Done.

**Pull request merged**